cache/src/main/kotlin/com/dropbox/android/external/cache4/Cache.kt

```diff
@@ -54,6 +54,14 @@ interface Cache<in Key : Any, Value : Any> {
      */
     interface Builder {
 
+        @ExperimentalTime
+        var expireAfterWriteDuration: Duration
```

Comment:
had to pull these into interface for extension build function.  My question about these (for follow up) - do we need the setter functions or can we have a builder with only vars and `set()`

Reply:
Setter functions return the instance whereas simple vars do not. So that allows chain of the method calls.